test/SILOptimizer/pass_printer.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-swift-frontend -Xllvm -sil-print-before=definite-init -emit-sil %s -o /dev/null 2>&1 | %FileCheck --check-prefix=BEFORE %s
+// RUN: %target-swift-frontend -Xllvm -sil-print-after=definite-init -emit-sil %s -o /dev/null 2>&1 | %FileCheck --check-prefix=AFTER %s
+// RUN: %target-swift-frontend -Xllvm -sil-disable-pass=definite-init -Xllvm -sil-print-pass-name -emit-sil %s -o /dev/null 2>&1 | %FileCheck --check-prefix=DISABLE %s
+
+// BEFORE: SIL function before Guaranteed Passes Definite Initialization for Diagnostics (definite-init) (#1)
```

Comment:
Actually question. I forgot, does this work when assertions are disabled? You may want a requires line in such a case.